sdk/search/azure-search-documents/azure/search/documents/_service/_search_service_client_base.py

```diff
@@ -0,0 +1,57 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+from typing import TYPE_CHECKING
+
+from .._headers_mixin import HeadersMixin
+from ._datasources_client import SearchDataSourcesClient
+from ._indexes_client import SearchIndexesClient
+from ._indexers_client import SearchIndexersClient
+from ._skillsets_client import SearchSkillsetsClient
+from ._synonym_maps_client import SearchSynonymMapsClient
```

Comment:
Don't seem those imports are necessary here?

Reply:
how is pylint passing? ⭕

Reply:
removed these - but im shocked to see pylint not catching unused-import